provider/aws/dist/rack.json

```diff
@@ -1680,16 +1680,16 @@
       },
       "Type": "AWS::EC2::SecurityGroup"
     },
-    "RackWeb": {
-      "DependsOn": [ "RackWebTasks", "Balancer", "Cluster", "CustomTopic", "Instances", "ServiceRole" ],
+    "WebService": {
```

Comment:
Could we call this RackAPI just so it's a little easier to know what it is if looking at the Services list in ECS?